upi/openstack/inventory.yaml

```diff
@@ -73,8 +66,8 @@ all:
       # nodes is zero
       os_master_schedulable: "{{ os_compute_nodes_number | int == 0 }}"
 
-      # Name of the IPv6 subnet. Uncomment to enable dual-stack support
-      #os_subnet6: ocp-subnet-v6
+      # Uncomment to enable dual-stack support
+      #dualstack: true
```

Comment:
why do we need to change this?
My thinking is that we would like to avoid additional changes given this fix will be backported.

Reply:
I have't found a way to read vars from external file in inventory, and the name of the subnet have random name now (which is the clue of this patch), so that I needed to use something explicit for enabling dualstack.